app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class UsersController < ApplicationController
+  def new
+      @user = User.new
+      if logged_in?
+        redirect_to tasks_path
+      end
```

Comment:
elseのない条件式は後置ifの構文で記載すると、コード量が減り、else部分の条件を考慮漏れしていないことを明示できます。
```suggestion
    redirect_to tasks_path  if logged_in?
```